File: 10013/super.c
#include <stdio.h>

// Accepted 

int num1[1000005];
int num2[1000005];
int sum[1000005];
int n;
int t;

int main(int argc, char *argv[])
{
	int i, carry, x;

	scanf("%d", &t);

	while(t--)
	{
		scanf("%d", &n);

		for(i=0; i<n; i++)
			scanf("%d %d", &num1[i], &num2[i]);

		carry = 0;
		for(i=(n-1); i>=0; i--)
		{
			x = num1[i] + num2[i] + carry;

			sum[(n - 1) - i] = x % 10;
			carry = x / 10; 
		}

		if (carry > 0)
		{
			sum[n] = carry;
			printf("%d", carry);
		}


		for(i=(n-1); i>=0; i--)
			printf("%d", sum[i]);

		printf("\n");

		if(t > 0)
			printf("\n");
	}

	return 0;
}

File: 10013/super.py
import sys

t = int(sys.stdin.readline())

for i in range(t):
	line = sys.stdin.readline()
	n = int(sys.stdin.readline())

	num1 = []
	num2 = []

	for j in range(n):
		line = sys.stdin.readline().strip().split()

		num1.append(int(line[0]))
		num2.append(int(line[1]))

	sum_num = []
	carry = 0
	for j in range(n-1, -1, -1):
		x = num1[j] + num2[j] + carry

		sum_num.append(x % 10)
		carry = x // 10

	if carry != 0:
		sum_num.append(carry)

	sum_num.reverse()

	y = [str(k) for k in sum_num]

	num = ''.join(y)

	if i > 0:
		print()

	print(num)
